ParkingApplication.api/Services/ParkingService.cs

```diff
@@ -12,31 +13,52 @@ public class ParkingService : IParkingService
 
     public int MaxSlots { get; set; }
 
+    private readonly ICommandRepository<ParkingModel> _commandRepository;
+    private readonly IQueryRepository<ParkingModel> _queryRepository;
+    private readonly IRepositoryFactory _repositoryFactory;
     private readonly IAccountService _accountService;
 
-    public ParkingService(IAccountService accountService)
+    public ParkingService(IAccountService accountService, IRepositoryFactory repositoryFactory)
     {
+        _repositoryFactory = repositoryFactory;
+        _queryRepository = repositoryFactory.CreateQueryRepository<ParkingModel>();
+        _commandRepository = repositoryFactory.CreateCommandRepository<ParkingModel>();
         _accountService = accountService;
         MaxSlots = 50;
+
+    }
+    public List<ParkingModel> GetAllCars()
+    {
+        return _queryRepository.GetAll();
+    }
+
+    public void AddParking(ParkingModel parkingModel)
+    {
+        _commandRepository.Create(parkingModel);
     }
 
-    public List<ParkingModel> GetParkedCar { get; } = new List<ParkingModel>  { new ParkingModel(1, "abc"), new ParkingModel(2, "abcde") };
+    public void RemoveParking(ParkingModel parkingModel)
+    {
+        _commandRepository.Delete(parkingModel);
+    }
 
-    public void AddToParking(int id, string carNumber)
+    public void AddToParking(int userId, string carNumber)
     {
-        if(GetParkedCar.Count < MaxSlots && carNumber != null)
+        var getCars = _queryRepository.GetAll();
+        if(getCars.Count < MaxSlots && carNumber != null)
         {
-            var Car = new ParkingModel(id, carNumber);
-            GetParkedCar.Add(Car);
+            var Car = new ParkingModel(userId, carNumber);
+            _commandRepository.Create(Car);
 
         }
     }
 
     public bool ExitParking(string carNumber)
     {
-        var parkedCar = GetParkedCar.Find(car => car.CarNumber == carNumber);
+        var getCars = _queryRepository.GetAll();
+        var parkedCar = getCars.Find(car => car.CarNumber == carNumber);
```

Comment:
you could move this into repository and have a GetByCarNumber method

Reply:
Done!